grpc_core/mix.exs

```diff
@@ -0,0 +1,61 @@
+defmodule GRPC.Core.MixProject do
+  use Mix.Project
+
+  @source_url "https://github.com/elixir-grpc/grpc"
+  @version "1.0.0"
```

Comment:
```suggestion
  @version "0.12.0"
```

Reply:
yeah, I think we should keep 0.12, or at least 1.0.0-rc.1 if we're really sure we're ready to bump. We won't be able to make breaking changes after the bump